ui/raidboss/common_replacement.js

```diff
@@ -9,6 +9,24 @@ const syncKeys = {
   engage: 'Engage!',
 };
 
+const textKeys = {
+  // Match directions in replaceText
+  // eg: `(N)`, `(SW)`, `(NE/NW)`, etc.
+  E: '(?<= \\(|\\/)E(?=\\)|\\/)',
+  N: '(?<= \\(|\\/)N(?=\\)|\\/)',
+  S: '(?<= \\(|\\/)S(?=\\)|\\/)',
+  W: '(?<= \\(|\\/)W(?=\\)|\\/)',
+  NE: '(?<= \\(|\\/)NE(?=\\)|\\/)',
+  NW: '(?<= \\(|\\/)NW(?=\\)|\\/)',
+  SE: '(?<= \\(|\\/)SE(?=\\)|\\/)',
+  SW: '(?<= \\(|\\/)SW(?=\\)|\\/)',
+  // Match Roles in replaceText
+  // eg: `(Tank)`, `(Healer)`, `(DPS)`, etc
+  Tank: '(?<= \\(|\\/)Tank(?=\\)|\\/)',
+  Healer: '(?<= \\(|\\/)Healer(?=\\)|\\/)',
+  DPS: '(?<= \\(|\\/)DPS(?=\\)|\\/)',
```

Comment:
Maybe it is not very useful to translate something like `(T)` and `(T/H)`?
So I have only added full names of roles.